libpgcommon/lwgeom_transform.c

```diff
@@ -220,19 +276,19 @@ static HTAB *CreatePJHash(void)
 	ctl.entrysize = sizeof(PJHashEntry);
 	ctl.hash = mcxt_ptr_hash;
 
-	return hash_create("PostGIS PROJ4 Backend projPJ MemoryContext Hash", PROJ4_BACKEND_HASH_SIZE, &ctl, (HASH_ELEM | HASH_FUNCTION));
+	return hash_create("PostGIS PROJ Backend MemoryContext Hash", PROJ_BACKEND_HASH_SIZE, &ctl, (HASH_ELEM | HASH_FUNCTION));
 }
 
-static void AddPJHashEntry(MemoryContext mcxt, projPJ projection)
+static void AddPJHashEntry(MemoryContext mcxt, PJ* projection)
 {
 	bool found;
-	void **key;
-	PJHashEntry *he;
+	void** key;
```

Comment:
Why did you change the position of the `*` in these functions? AFAIK (and clang-format applies) it's `type *name`

Reply:
I'm just OCD'ing on that lately, since pushed GEOS to type-bound in the big reformat.